buildspec.yml

```diff
@@ -0,0 +1,27 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay2&
+      - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
+  pre_build:
+    commands:
+    - echo Logging in to AWS ECR....
+    - aws --version
+    - aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/uktrade
```

Comment:
Can we set up an IAM role giving CodeBuild the required access instead of needing to log in?

If yes, I would be happy to see this happen in a follow up pull request.

Reply:
@WillGibson you still need a role to give you access, but you need to use the get-login command to authenticate.

Reply:
I don't think I understand how logging in to AWS ECR helps us with Docker Hub rate limiting, maybe our AWS ECR has a paid Docker Hub subscription or something?